apps/console/src/extensions/components/account-login/pages/username-validation-edit.tsx

```diff
@@ -233,7 +233,7 @@ export const UsernameValidationEditPage: FunctionComponent<UsernameValidationEdi
         const usernameConf: ValidationDataInterface[] =
             validationData?.filter((data: ValidationDataInterface) => data.field === "username");
 
-        if (usernameConf === undefined || usernameConf.length < 1) {
+        if (usernameConf === undefined || usernameConf?.length < 1) {
```

Comment:
We don't need the explicit undefined checking, as we are using optional chaining, do we?

```suggestion
        if (usernameConf?.length < 1) {
```

Reply:
Fixed with aa6b04cb1d9612a46456994d7aeccf49e7b02602